public/js/components/App.js

```diff
@@ -128,10 +134,12 @@ const App = React.createClass({
       { className: classnames("debugger theme-body",
                               { "theme-light": !isFirefoxPanel() }) },
       SplitBox({
-        initialWidth: 300,
+        width: this.props.sidebars.toJS().left.width,
+        resizeSidebar: this.props.resizeSidebar.bind(null, "left"),
```

Comment:
we've been following the wrapping pattern `(width) => this.props.resizebar("left", width)`